pendulum_control/README.md

```diff
@@ -19,6 +19,12 @@ The demo should work with both middleware implementations.
 A few command line arguments related to real-time performance profiling are provided by rttest.
 See https://github.com/ros2/rttest/blob/master/README.md for more information.
 
+The demo will spit out a lot of output as it runs. Try redirecting the results to a file:
+
+```
+./build/pendulum_control/pendulum_demo[__rmw_opensplice/__rmw_connext] > output.txt
```

Comment:
After sourcing install/setup.* you should use the executables in install/bin which is on the path already. The ones in build are intermediate products only. 

Also there's no need to build the unit tests for the demo. It will save time compiling. And I wouldn't recommend using `--only pendulum_control` since you do not know the state of the workspace and whether every dependency has already been compiled. (I made a new workspace and didn't have anything built yet).